frontend/src/components/Home/HomeWithoutLogin/HomeWithoutLogin.tsx

```diff
@@ -0,0 +1,26 @@
+import './HomeWithoutLogin.css';
+import {Link} from "react-router-dom";
+
+
+export default function HomeWithoutLogin() {
+    function login() {
+        const host = window.location.host === 'localhost:5173' ? 'http://localhost:8080': window.location.origin
+        window.open(host + '/oauth2/authorization/google', '_self')
+        console.log("I want to sign in with google");
```

Comment:
You should not use console.log in production code. Instead you can use a logger

Reply:
Thanks! I overlooked these console.logs. I replaced them with a Logger